src/MessageDispatch.KurrentDB/KurrentDbSubscriber.cs

```diff
@@ -73,13 +83,38 @@ private KurrentDbSubscriber(
         KurrentDBClient kurrentDbClient,
         IDispatcher<ResolvedEvent> dispatcher,
         string streamName,
-        ILogger logger)
-        => Init(kurrentDbClient, dispatcher, streamName, logger, liveOnly: true);
-
+        ILogger logger)
+    {
+        SetupCallbackTimer();
+
+        Init(kurrentDbClient, dispatcher, streamName, logger, liveOnly: true);
+    }
+
+    /// <summary>
+    /// Creates a callback timer to ping KurrentDB every 60s to stop it from going idle
+    /// </summary>
+    private void SetupCallbackTimer()
+    {
+        //Create the timer and set it for a sixty second interval
+        _timer = new Timer();
+        _timer.Interval = 60000;

```

Comment:
Not sure if this was on the list to change anyway but can we have a const for this.